src/utils/yarn.js

```diff
@@ -218,7 +220,8 @@ export async function userAgent() {
     localYarn,
     ['config', 'get', 'user-agent'],
     {
-      tty: false
+      tty: false,
+      silent: true
```

Comment:
Whats this for?

Reply:
It hides the output of the user agent call we make in order to populate our own bolt user agent environment variable.
E.g. from our a/f build step:

```$ /opt/atlassian/pipelines/agent/build/node_modules/bolt/node_modules/.bin/yarn config get user-agent 
$ /opt/atlassian/pipelines/agent/build/node_modules/bolt/node_modules/.bin/yarn config get user-agent yarn/1.17.3 npm/? node/v12.14.1 linux x64
$ /opt/atlassian/pipelines/agent/build/node_modules/bolt/node_modules/.bin/yarn config get user-agent yarn/1.17.3 npm/? node/v12.14.1 linux x64
$ /opt/atlassian/pipelines/agent/build/node_modules/bolt/node_modules/.bin/yarn config get user-agent yarn/1.17.3 npm/? node/v12.14.1 linux x64
$ /opt/atlassian/pipelines/agent/build/node_modules/bolt/node_modules/.bin/yarn config get user-agent yarn/1.17.3 npm/? node/v12.14.1 linux x64
$ /opt/atlassian/pipelines/agent/build/node_modules/bolt/node_modules/.bin/yarn config get user-agent yarn/1.17.3 npm/? node/v12.14.1 linux x64
$ /opt/atlassian/pipelines/agent/build/node_modules/bolt/node_modules/.bin/yarn config get user-agent yarn/1.17.3 npm/? node/v12.14.1 linux x64
```